src/main/java/com/powsybl/balances_adjustment/util/CountryArea.java

```diff
@@ -60,6 +61,26 @@ public double getNetPosition() {
         return Collections.unmodifiableCollection(busesCache);
     }
 
+    public double getLeavingFlowToCountry(Country country) {
```

Comment:
What about using CountryArea in the signature instead of Country to remain consistent with the global framework ? Then you could use isAreaBorder method of the CountryArea in argument, it might be a bit more concise.

Reply:
That was just an idea. For us it is perfectly good to have Country so it's ok if no modification.